dev_guide/daemonsets.adoc

```diff
@@ -19,8 +19,36 @@ A daemonset can be used to run replicas of a pod on specific or all nodes in an
 Use daemonsets to create shared storage, run a logging pod on every node in
 your cluster, or deploy a monitoring agent on every node.
 
+For security reasons and by their nature creating DaemonSets is restricted to cluster administrators.
+
 For more information on daemonsets, see the link:http://kubernetes.io/docs/admin/daemons/[Kubernetes documentation].
 
+[IMPORTANT]
+====
+Before you start using daemonset you need to make sure to disable project default node selector in your namespace.
+
```

Comment:
s/daemonset/DaemonSets

Before you start using DaemonSets ensure that you disable the default project-wide xref:../admin_guide/managing_projects.adoc#using-node-selectors[node selector] in your namespace.

Reply:
s/daemonset/DaemonSets

I'd prefer it to be this way but the rest of the doc uses all lowercase.

Reply:
@tnozicka I agree. The term is used both ways throughout the full docs set. But, we should be consistent in this topic at least.

Reply:
maybe we can get rid of `you need to make sure` so the sentence becomes `Before you start using daemonsets, disable the default project-wide...`